BUILDING.md

```diff
@@ -253,6 +253,16 @@ $ ./node ./test/parallel/test-stream2-transform.js
 Remember to recompile with `make -j4` in between test runs if you change code in
 the `lib` or `src` directories.
 
+The tests attempt to detect support for IPv6 and exclude IPv6 tests if
+appropriate. However, if your main interface has IPv6 addresses then your
+loopback interface must also have '::1' enabled. For some default installations
+on Ubuntu that does not seem to be the case. '::1' can be enabled on the
+loopback interface on Ubuntu using:
```

Comment:
Replace the last sentence with this?

```markdown
To enable '::1' on the loopback interface on Ubuntu:
```

Reply:
Or at least 
```
You can enable...
```

Reply:
I'm sure I had something like "You can .." but guidance is not to use you. :)